newdle/client/src/components/answer/AnswerPage.js

```diff
@@ -200,6 +211,9 @@ export default function AnswerPage() {
                 />
               </Segment>
             )}
+            <Header as="h3" className={styles.date}>
+              Displayed timezone: {userTz}
```

Comment:
I think it'd be important to show the newdle's timezone as well, so the user knows that the event the newdle is about is taking place in timezone X, but the times are shown to them in timezone Y.

Reply:
Good idea, thanks!

Reply:
Added a simple "Newdle timezone" header, will update the PR's preview image ASAP